mariadb_skysql/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bed39b01-bbc5-4819-9744-7cf4a3d90f6d",
+  "app_id": "mariadb-skysql",
```

Comment:
I'm not sure why this check is flagging that `mariadb-skysql` is not working.

Reply:
Not sure if this was for something else, but the error message above should go away when @emarsha94 submits the request to add the new logo.